docs/FAQ/FAQ-Wallet.md

```diff
@@ -90,6 +90,10 @@ Using different derivation schemes with your xpub, you can also choose to create
 |Multi-sig P2SH-P2WSH	| 2-of-xpub1...-xpub2...-[p2sh] |
 |Multi-sig P2SH |	2-of-xpub1...-xpub2...-[legacy] |
 
+:::tip
+On top of the Xpub extended public key formats shown above, BTCPay Server supports Ypub and Zpub formats. Please note that these will be converted to Xpub once the wallet setup is completed. This has no effect on how you receive or send funds.
```

Comment:
These are usually spelled with lowercase first letter, or all uppercase  so this is for consistency with other sources, haven't seen first uppercase as much compared to the two.

```suggestion
On top of the xPub extended public key formats shown above, BTCPay Server supports yPub and zPub formats. Please note that these will be converted to xPub once the wallet setup is completed. This has no effect on how you receive or send funds.
```